x-pack/plugins/actions/server/saved_objects/mappings.json

```diff
@@ -45,6 +45,10 @@
       "relatedSavedObjects": {
         "enabled": false,
         "type": "object"
+      },
+      "source": {
```

Comment:
Adding `source` field but not indexing it because we don't need to query on it. Also not adding a migration because this is not a required field.

Reply:
Do you really need to add it explicitly to the mappings then?

Reply:
I thought we had `dynamic: strict` for saved object mappings so we had to identify all the fields even if we didn't need to map them? If that's not the case, I am happy to remove!

Reply:
We're using `dynamic:strict` at the root level, but you can add `dynamic: false` at the top level of your type's mapping to override it. I thought it was already the case for response ops types tbh.

Reply:
It looks like that is the case for the `cases` SO type but not for `action`, `action_task_params` or `alert` SO types. Is that the preferred method? I can add it to `action_task_params` in this PR and create a separate PR for the other SO types if so

Reply:
I think check w/ @XavierM - he's been working on fine-tuning the mappings for dynamic false; he probably has a feel whether we should do these adhoc or wait and do them all at once. 

We **did** recently just add a `dynamic: false` to an "inner" struct in the rule SO, and I posted some comments on it. https://github.com/elastic/kibana/pull/148038/files#r1091213922

Reply:
> he probably has a feel whether we should do these adhoc or wait and do them all at once

Doing it in another PR works for me, as long as we make sure this new field is in the radar of the other PR/PR author.

I'll approve to unblock and will let you figure out how/when you want to do the field removal.

Reply:
Discussed with Xavier and I will add `dynamic:false` to the `action_task_params` mappings in this PR, he will make note of the new source field in his PR: https://github.com/elastic/kibana/pull/152857 which will also set dynamic:false for the other Response Ops SO types.